src/library_wasm_worker.js

```diff
@@ -139,7 +139,7 @@ mergeInto(LibraryManager.library, {
       'sb': stackLowestAddress, // sb = stack bottom (lowest stack address, SP points at this when stack is full)
       'sz': stackSize,          // sz = stack size
     });
-    worker.addEventListener('message', __wasm_worker_runPostMessage);
+    worker.onmessage = __wasm_worker_runPostMessage;
```

Comment:
`addEventListener` doesn't work under node?

Reply:
Our polyfill in `src/shell.js` does not support this no.  We could add to the polyfill, but probably better to be consistent in how we attach the onmessage handler (this is how we do it in the pthread code)

Reply:
Using addEventListener here was very deliberate, because addEventListener can be used many times, but there can be only one `.onmessage`, so that was reserved for the end user code to utilize. In the past we have had bug reports about "please be a good composable library" about this, which is why I have taken the stance of not using .onxxx on any possible publicly accessible event handlers.

.onmessage is shorter though. so I wouldn't mind it from code golfing perspective, but I'd prefer addEventListener for safety.

Reply:
Out of interest if we use `.onmessage` here can our users stil used `addEventListenter()`?

I feel like we should be consistent between wasm workers and pthreads there.   I don't mind changing both but we might also need to improve the worker polyfill that we have for node